bindings/wasm/package.json

```diff
@@ -3,6 +3,9 @@
   "version": "2.5.1",
   "description": "Geometry library for topological robustness",
   "main": "manifold.js",
+  "scripts": {
+    "docs": "npx typedoc"
```

Comment:
Hmm, are all the extra files here created automatically by running this command? If so, I think instead of checking them in to the repo, we should just tweak our Github Actions CI to run this script as part of `deploy-documentation` and copy them straight into our served Pages. Would that be reasonable?

Reply:
@elalish I was not sure what do you prefer but sure, instead of checking them in to the rep we could move them to GitHub Action CI. Would you like me to do it, or would you prefer to handle it? I'm not yet familiar with your file organization.

Btw I've resolved CLA and removed bindings/wasm/docs folder from the repo.